src/Entity/Hashtag.php

```diff
@@ -5,28 +5,37 @@
 class Hashtag extends AbstractEntity
 {
     /**
-     * {@inhertDoc}.
```

Comment:
The `getEntityKey()` method is private to this class and not within the abstract. I think it's just GitHub trying to be clever about the changes when really this whole docblock & method are new, not replacements of previous.